### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a-eid**
- option to stay on top of UIWindow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,40 @@ const styles = StyleSheet.create({
 
 ```
 
+To make the compoent stay  on top even after pushing modals, you need to add the following code to your `appDelegate.m`
```
Comment (WoLewicki) [inline README.md:50]:
```suggestion
To make the compoent stay on top even after pushing modals, you need to add the following code to your `appDelegate.m` 
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,40 @@ const styles = StyleSheet.create({
 
 ```
 
+To make the compoent stay  on top even after pushing modals, you need to add the following code to your `appDelegate.m` 
+
+```swift
```
Comment (WoLewicki) [inline README.md:52]: Shouldn't it be objc?

**Review by WoLewicki — CHANGES_REQUESTED** [review]
Thanks for this PR! Good work, I left some comments regarding indents etc.

File: README.md (hunk 2 above)
Comment (a-eid) [reply README.md:52]: yes :).

**Pull request merged**